ddclient.in

```diff
@@ -3454,7 +3454,8 @@ sub get_ipv6 {
         $ipv6 = get_ip_from_interface($arg, 6);
     }  elsif ($p{'usev6'} eq 'cmdv6' || $p{'usev6'} eq 'cmd') {
         ## Obtain IPv6 address by executing the command in "cmdv6=<command>"
-        warning("'--cmd-skip' ignored") if opt('verbose') && p{'cmd-skip'};
+        warning("'--cmd-skip' ignored for '--usev6=$p{'usev6'}'")
+            if (opt('verbose') && defined($p{'cmd-skip'}));
```

Comment:
Style nits: Unwrap the line (it's shorter than 99 columns unwrapped) and drop the parentheses for consistency:

```suggestion
        warning("'--cmd-skip' ignored for '--usev6=$p{'usev6'}'") if opt('verbose') && defined($p{'cmd-skip'});
```

Reply:
Updated, and removed the extra parenthesis. The unwrapped line exceeds 100 columns though - am I miscalculating?

Reply:
Whoops, you're right.  I coulda swore that it was shorter than 100.  My bad.

Reply:
😆 Should I wrap it back or keep it unwrapped this way?